_sass/_uswds-theme-overrides.scss

```diff
@@ -204,6 +204,7 @@ $theme-button-border-radius: "sm";
 //
 // Uses https://civicactions.github.io/uswds-color-tool/ to find the closest color to `$c-dhs-blue`
 $theme-identifier-background-color: "blue-cool-60v";
+$theme-identifier-identity-domain-color: "base-lightest";
```

Comment:
4. I ran the [axe accessibility browser plugin](https://www.deque.com/axe/) and an issue was flagged regarding the domain name contrast on Identifier. This was probably due to our change to grays above. We can easily fix this by updating a default identifier setting for domain color.